src/pages/settings/Payments/PaymentMethodList.js

```diff
@@ -60,6 +60,12 @@ const propTypes = {
         walletLinkedAccountType: PropTypes.string,
     }),
 
+    /** Type of active/highlighted payment method */
+    actionPaymentMethodType: PropTypes.oneOf([CONST.PAYMENT_METHODS.DEBIT_CARD, CONST.PAYMENT_METHODS.BANK_ACCOUNT, '']),
+
+    /** ID of active/highlighted payment method */
+    activePaymentMethodID: PropTypes.number,
```

Comment:
I noticed this error in the console log:

<img width="500" alt="Screen Shot 2022-01-25 at 11 31 06" src="https://user-images.githubusercontent.com/6829422/151028809-088ee949-1252-43f9-8b7b-aa91f57bcbc8.png">

I think we'll need to change the type of the property to support both numbers and strings to something like this:

```suggestion
    activePaymentMethodID: PropTypes.oneOfType([
          PropTypes.string,
          PropTypes.number,
    ]),
```